ui/helpers/urlHelper.js

```diff
@@ -142,3 +143,11 @@ export const bugzillaBugsApi = function bugzillaBugsApi(api, params) {
   const query = createQueryParams(params);
   return `https://bugzilla.mozilla.org/${api}${query}`;
 };
+
+export const getWriteHeaders = function getWriteHeaders() {
+  return new Headers({
+    'X-CSRFToken': Cookies.get('csrftoken'),
```

Comment:
I think we can send the token as a cookie rather than a header? (AngularJS $http only supported the latter, but that wasn't great, since it meant we couldn't enable the `httpOnly` directive making the cookie more secure)

The fetch call will need a top level `credentials: 'same-origin'` option setting.

See:
https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch#Parameters

This would mean we can drop the js-cookie dependency too.

It might mean that instead of the `getWriteHeaders()` helper, a slightly higher level `postJson()` helper might be useful?